db/types/money.py

```diff
@@ -25,3 +29,18 @@ def install(engine):
         conn.execute(text(drop_domain_query))
         conn.execute(text(create_domain_query))
         conn.commit()
+
+
+def get_money_array(table_oid, engine, column_attnum):
```

Comment:
I think you should rename this or add some clear comments that this only gets an array representing the pieces of the string in the first row. It's a bit confusing, and we want to make sure it's clear that the currency display option decisions are based on a single entry.